spiceypy/tests/test_spiceerrors.py

```diff
@@ -53,3 +53,19 @@ def test_foundErrorChecker():
     with pytest.raises(spice.stypes.SpiceyError):
         spice.bodc2n(-9991)
     spice.reset()
+
+def test_disable_found_catch():
+    with spice.no_found_check():
+        name, found = spice.bodc2n(-9991)
+        assert not found
+    spice.reset()
```

Comment:
Not sure spice.reset() is needed; I don't think this results in spice.failed() being True.